site/src/components/layout.md

```diff
@@ -136,6 +140,76 @@ The specification for this layout is as follows.
 | Desktop    | Menu, main, aside are side by side. Main is wider than menu.                                                                  |
 | Widescreen | Same as desktop with scaling gutter that keeps combined width of menu, main, aside to the width of the widescreen breakpoint. |
 
+### Sidecar left layout
+
+A "sidecar" is a smaller companion container that sits beside the main content container on tablet screens and wider. In `layout-sidecar-left` the sidecar refers to the `layout-body-menu` element, which sits to the left of `layout-body-main`. Unlike other layouts, this layout does not allow the usage of the `layout-body-aside` containers.
+
+How do I apply it? `layout-sidecar-left` to the `layout` element.
+Required elements: all except `layout-body-hero` and `layout-body-aside` (see allowed elements).
+Allowed elements: all except `layout-body-aside`.
+
+The following block is arranged from narrow widths on the left to wider widths on the right.
+
+```txt
```

Comment:
Would it be helpful to label these blocks to supplement the previous sentence? Something like this?

![image](https://github.com/microsoft/atlas-design/assets/94572161/267ba211-6504-4fc1-9d55-9a19071ae310)

Reply:
Nothing wrong with some better labelling. Updated all these text blocks on this page.